example/app.jsx

```diff
@@ -85,7 +85,10 @@ class App extends React.Component {
               cropRadius={50}
               onCrop={this.onCrop}
               onClose={this.onClose}
+              lineWidth={5}
               src={this.state.src}
+              changeAspecRatio
```

Comment:
Minor: We can also remove the `width: '150px'` styling from the `img` tag so that the preview is not skewed